docs/sources/operators-guide/visualizing-metrics/dashboards/scaling/index.md

```diff
@@ -7,7 +7,8 @@ weight: 140
 
 # Grafana Mimir Scaling dashboard
 
-The Scaling dashboard shows services that are not scaled correctly.
+The Scaling dashboard suggests services that should be scaled up in the event of a failure for one of several specific reasons.
```

Comment:
```suggestion
The scaling dashboard displays services that you can scale up, in the event of a failure for one or more specific reasons.
```
I like this comma here because it separates two preposition "up in".